install.sh

```diff
@@ -7,6 +7,20 @@ if [[ -n "$MSYSTEM" ]]; then
   exit 1
 fi
 
+# Check if script was executed with root/sudo
+if [ "$CI" != "true" ]; then
+  if [ "$(id -u)" -ne 0 ]; then
+    echo "You should be using sudo, are you sure you want to continue? [y/n]"
+    read -r answer
+    if [ "$answer" != "${answer#[Yy]}" ] ;then
```

Comment:
Not sure if in misinterpreting this but is the logic flipped here?

Reply:
i flipped it